### PR TITLE
Donate - BLM

### PR DESCRIPTION
**Pull request by Akhan221**
### Changes Made

Updated BLM donate page with appropriate organizations and content using card stack. Replacement image may be needed for Mutual Aid Network card.

### CODE REVIEW

File: src/components/blm/index.jsx [hunk 1]
```diff
@@ -25,36 +19,28 @@ class BLMPages extends Component {
             <BLMDirectory />
           </Route>
 
-          <Route path="/blm/education">
-            BLM Education
-          </Route>
+          <Route path="/blm/education">BLM Education</Route>
```
Comment (amanmibra) [inline src/components/blm/index.jsx:22]: Dont change the spacing of the Route components please

**Comment by amanmibra** [comment]
Also this is a good opportunity to pull in changes from master (something you should do right before you merge your PR)

Simply call `git pull origin master` and fix any merge conflicts

File: src/components/blm/index.jsx (hunk 1 above)
Comment (Akhan221) [reply src/components/blm/index.jsx]: Got it, just fixed that

File: src/components/blm/index.jsx [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
-} from 'semantic-ui-react';
+import { Container, Header } from "semantic-ui-react";
 
 // router
-import {
-  Switch,
-  Redirect,
-  Route,
-} from 'react-router-dom';
+import { Switch, Redirect, Route } from "react-router-dom";
```
Comment (amanmibra) [inline src/components/blm/index.jsx:7]: Small thing. Please see how we import other components and how they are multi-line if more than one component is imported.

**Pull request merged**